src/Driver/DriverAllocator.php

```diff
@@ -28,7 +28,7 @@ public static function fromPDO(\PDO $PDO): DriverInterface
         if (strpos($rawVersion, 'MariaDB') !== false) {
             $type = self::TYPE_MARIA_DB;
             $split = explode('-', $rawVersion);
-            $version = $split[1];
+            $version = $split[0];
```

Comment:
Thanks for the great pull request! 🎉
Please fix this to maintain compatibility with MariaDB version 10.
```suggestion
            if (strtolower($split[1]) === 'mariadb') {
                // MariaDB 11 ~
                $version = $split[0];
            } else {
                $version = $split[1];
            }
```
I'll release it as soon as it's fixed.